pkg/api/common/helper_test.go

```diff
@@ -9,18 +9,18 @@ func Test_GetValidPatchVersion(t *testing.T) {
 	}
 
 	version = GetValidPatchVersion(Kubernetes, "1.6.3")
-	if version != KubernetesVersion1Dot6Dot11 {
-		t.Errorf("It is not Kubernetes version %s", KubernetesVersion1Dot6Dot11)
+	if version != KubernetesVersion1Dot6Dot12 {
+		t.Errorf("It is not Kubernetes version %s", KubernetesVersion1Dot6Dot12)
 	}
 
 	version = GetValidPatchVersion(Kubernetes, "1.7.3")
-	if version != KubernetesVersion1Dot7Dot9 {
-		t.Errorf("It is not Kubernetes version %s", KubernetesVersion1Dot6Dot11)
+	if version != KubernetesVersion1Dot7Dot10 {
+		t.Errorf("It is not Kubernetes version %s", KubernetesVersion1Dot7Dot10)
 	}
 
 	version = GetValidPatchVersion(Kubernetes, "1.8.2")
 	if version != KubernetesVersion1Dot8Dot2 {
-		t.Errorf("It is not Kubernetes version %s", KubernetesVersion1Dot6Dot11)
```

Comment:
@JackQuincy can you confirm this is intended? It looks to me this was returning an error message that wasn't consistent with the version it was expecting. Same thing for other cases in this test function.

Reply:
was just a copy paste bug, thanks for fixing